Server/ReasnAPI/ReasnAPI/Mappers/EventMapper.cs

```diff
@@ -0,0 +1,54 @@
+﻿using ReasnAPI.Models.Database;
+using ReasnAPI.Models.DTOs;
+
+namespace ReasnAPI.Mappers
+{
+    public static class EventMapper
+    {
+        public static EventDto ToDto(this Event eventToMap)
+        {
+            var tags = eventToMap.Tags
+                .Select(t => new TagDto { Name = t.Name })

```

Comment:
Wouldn't be easier to use already created function in TagMapper `ToDtoList()`?

Reply:
changed